WordPress/Tests/NamingConventions/ValidVariableNameUnitTest.php

```diff
@@ -76,7 +76,9 @@ public function getErrorList() {
 	 * @return array <int line number> => <int number of warnings>
 	 */
 	public function getWarningList() {
-		return array();
+		return array(
+			1   => 1, // Warning about property setting.
```

Comment:
As it is only a single value I would have only have a single space between the key and `=>`